src/cli-validator/utils/fixProblems.js

```diff
@@ -0,0 +1,229 @@
+const each = require('lodash/each');
```

Comment:
@dpopp07 Should we have copyright stmts in the validator files?

Reply:
Yes, probably. I don't think many, if any, have them though.